controllers/progressivesync_controller_test.go

```diff
@@ -724,6 +724,36 @@ var _ = Describe("ProgressiveRollout Controller", func() {
 			expected := ps.NewStatusCondition(syncv1alpha1.CompletedCondition, metav1.ConditionTrue, syncv1alpha1.StagesCompleteReason, "All stages completed")
 			ExpectCondition(&ps, expected.Type).Should(HaveStatus(expected.Status, expected.Reason, expected.Message))
 
+			Eventually(func() error {
```

Comment:
Can we add a comment here about why we're extending the test with these assertions?